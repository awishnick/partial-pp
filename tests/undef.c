// RUN: -UFOO

#ifdef FOO
// this will be removed
#endif

#ifdef BAR
// this will not be removed
#endif
