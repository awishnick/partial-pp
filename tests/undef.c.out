// RUN: -UFOO


#ifdef BAR
// this will not be removed
#endif
